charts/pandemi/values.yaml

```diff
@@ -0,0 +1,30 @@
+replicaCount: 1
+
+image:
+  repository: kitabisa/pandemi
+  pullPolicy: Always
+  tag: "latest"
+
+namespace: "global"
+
+serviceAccount:
+  annotations: {}
+  name: "pandemi-service-account"
+
+clusterRole:
+  apiGroups:
+    - ""
+  resources:
+    - "configmaps"
+    - "secrets"
+    - "namespaces"
+  verbs:
+    - "get"
+    - "watch"
+    - "list"
+    - "create"
+    - "update"
+    - "patch"
+
+clusterRoleBinding:
+  clusterRoleName: "pandemi-cluster-role"
```

Comment:
@budiariyanto mungkin lebih enak variablenya langsung gini aja mas

```
clusterRoleName: "pandemi-cluster-role"
```

instead of 
```
clusterRoleBinding:
  clusterRoleName: "pandemi-cluster-role"
```

Reply:
ini aku jadiin cluster role namenya dikeluarin jadi variable aja ya. Jadi bisa direfer sama cluster role binding